src/components/Index.tsx

```diff
@@ -78,6 +80,52 @@ const Index: React.FC = () => {
     return data;
   };
 
+  const fetchBirdData: FetchBirdData = async (birdName) => {
+    try {
+      // First try with bird name
+      const response = await fetch(
+        `https://en.wikipedia.org/api/rest_v1/page/summary/${birdName}`
+      );
+      if (response.ok) {
+        const data = await response.json();
+        if (data.type !== "disambiguation") {
+          return data;
+        }
+      }
+
+      // Try with "(bird)" suffix if first response was disambiguation
+      const speciesResponse = await fetch(
+        `https://en.wikipedia.org/api/rest_v1/page/summary/${birdName}_(bird)`
+      );
+      if (speciesResponse.ok) {
+        const data = await speciesResponse.json();
+        if (Object.keys(data).length > 0) {
+          return data;
+        }
+      }
+
+      // Try with "Common" prefix
+      const commonResponse = await fetch(
+        `https://en.wikipedia.org/api/rest_v1/page/summary/Common_${birdName}`
+      );
+      if (commonResponse.ok) {
+        return await commonResponse.json();
+      }
+
+      // Try with "European" prefix
+      const europeanResponse = await fetch(
+        `https://en.wikipedia.org/api/rest_v1/page/summary/European_${birdName}`
+      );
+      if (europeanResponse.ok) {
+        return await europeanResponse.json();
+      }
+
+    } catch (error) {
+      console.error("Error fetching bird data from Wikipedia:", error);
+      return {};
```

Comment:
it should probably throw error and if there is error than show error boundary component or alert, it couldnt download bird wiki data